.github/workflows/ci.yml

```diff
@@ -0,0 +1,14 @@
+name: CI
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: php-actions/composer@v5
+      - name: PHPUnit Tests
+        uses: php-actions/phpunit@v2
```

Comment:
Still not sure why this `uses` does not need a `-` but it works.

Reply:
@M4tini name is optional. The order does not matter, they are equal to the following json:
```json
{
    "name": "PHPUnit Tests",
    "uses": "php-actions/phpunit@v2"
}
```